StackCalculator/StackCalculatorProj/Stacks/Impl/StackLinkedList.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace StackCalculatorProj.Stacks.Impl;
+
+/// <summary>
+/// Stack class based on Linked List
+/// </summary>
+public class StackLinkedList <T> : IStack<T>
+{
+    private Node? _top;
+    private int _size = 0;
+    public int Size => _size;
```

Comment:
```suggestion
    public int Size { get; };
```
Компилятор буквально это и сделает, сгенерит поле и геттер, возвращающий его значение.